packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -553,6 +564,13 @@ class PageView extends StatefulWidget {
   /// children because constructing the [List] requires doing work for every
   /// child that could possibly be displayed in the page view, instead of just
   /// those children that are actually visible.
+  ///
+  /// {@template flutter.widgets.pageView.allowImplicitScrolling}
+  /// The [allowImplicitScrolling] parameter must not be null. If true, the
+  /// [PageView] will participate in accessibility scrolling more like a,
+  /// [ListView], where each scroll action will move to the next page rather
```

Comment:
Not sure what you mean by this. It should just enable implicit a11y scrolling (see my comment above).